src/com/oltpbenchmark/CommandLineOptions.java

```diff
@@ -0,0 +1,183 @@
+package com.oltpbenchmark;
+
+import org.apache.commons.cli.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class CommandLineOptions {
+    private CommandLine argsLine;
+
+    private static Options COMMAND_LINE_OPTS = new Options();
+    static {
+        COMMAND_LINE_OPTS.addOption("h", "help", false, "Print this help");
+        COMMAND_LINE_OPTS.addOption(
+                null,
+                "create",
+                true,
+                "Initialize the database for this benchmark");
+        COMMAND_LINE_OPTS.addOption(
+                null,
+                "clear",
+                true,
+                "Clear all records in the database for this benchmark");
+        COMMAND_LINE_OPTS.addOption(
+                null,
+                "load",
+                true,
+                "Load data using the benchmark's data loader");
+        COMMAND_LINE_OPTS.addOption(
+                null,
+                "execute",
+                true,
+                "Execute the benchmark workload");
+
+
+        COMMAND_LINE_OPTS.addOption(
+                "c",
+                "config",
+                true,
+                "Workload configuration file [default: config/workload_all.xml]");
+
+        COMMAND_LINE_OPTS.addOption("v", "verbose", false, "Display Messages");
+        COMMAND_LINE_OPTS.addOption("s", "sample", true, "Sampling window");
+        COMMAND_LINE_OPTS.addOption("im", "interval-monitor", true,
+                "Throughput Monitoring Interval in milliseconds");
+        COMMAND_LINE_OPTS.addOption("ss", false, "Verbose Sampling per Transaction");
+        COMMAND_LINE_OPTS.addOption("ts", "tracescript", true, "Script of transactions to execute");
```

Comment:
we don't use these.
Should we rmove them?

Reply:
we do use interval-monitor, but removing others